atl_msgs/PIDSettings roll_controller
atl_msgs/PIDSettings pitch_controller
atl_msgs/PIDSettings throttle_controller
float32 hover_throttle

atl_msgs/PIDSettings vx_controller
atl_msgs/PIDSettings vy_controller
atl_msgs/PIDSettings vz_controller
